src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const el in data) {
+    const li = document.createElement('li');
+
+    li.textContent = el;
+    list.append(li);
+
+    if (Object.keys(data[el]).length > 0) {
+      createTree(li, data[el]);
```

Comment:
It would be great to create a variable for `data[el]` to make your code a bit cleaner